app/Policies/CustodianPolicy.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Custodian;
+use App\Models\User;
+
+class CustodianPolicy
+{
+    public function viewAny(User $user): bool
+    {
+        return $user->inGroup([
+            User::GROUP_ADMINS,
```

Comment:
Just a note, the ADMINS user group is internal to us, not actual users. Those are set as admins via flags. Where as this gives direct access to the admin panel within soursd.